pseudoanonymize/anonymization.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Dict
+
+from .base import BaseProcessor
+from .common import make_chat_completion
+from .exceptions import UnparsableLLMOutputException
+
+with open('./prompts_examples/cot_example.txt', 'r') as file:
+    cot_demo = file.read()
+
+CoT_prompt = f"""
```

Comment:
nit: I'd default to writing acronyms out in full `chain_of_thought_prompt` - just so nobody ever wastes time wondering what they mean.

Reply:
Done